design/ceph/ceph-pull-realm.md

```diff
@@ -0,0 +1,94 @@
+
+For more information on Ceph multsite, please read the [ceph-realm](/design/ceph/ceph-realm.md) design document or [Ceph's documentation](http://docs.ceph.com/docs/master/radosgw/multisite/) on the feature.
+
+## Ceph Pull Realm Walkthrough
+
+### Pulling the Realm from another Ceph Cluster
+
+#### Config
+
+When the storage admin is ready to sync data from a Ceph cluster with multisite set up (primary cluster) to another Rook Ceph cluster (pulling cluster), the pulling cluster needs to have information about the primary cluster's realm and zone group.
+
+To do this, the storage admin needs to create a ceph-pull-realm resource on the pulling cluster with the same name as the realm from the primary cluster.
+
+The endpoint in the `pullEndpoint` section is an endpoint of an object-store in the master zone of the realm. 
+
+This endpoint must be resolvable from the pulling cluster.
+
+This example `ceph-pull-realm.yaml`, pulls `my-realm` from a primary Ceph cluster with endpoint http://my-realm-zone-b-objectstore-1:80 in its master zone.
+```yaml
+apiVersion: ceph.rook.io/v1alpha1
+kind: CephPullRealm
+metadata:
+  name: my-realm
+  namespace: rook-ceph-2
+spec:
+  pullEndpoint: http://my-realm-zone-b-objectstore-1:80
+```
+
+Now create the ceph-pull-realm.
+```bash
+kubectl create -f ceph-pull-realm.yaml
+```
+
+Once the creation of the ceph-pull-realm resource has been kicked off, the realm will be pulled on the pulling cluster. 
+
+If this realm pull succeeds, ceph-realm and ceph-zone-group resources will automatically be created. 
+
+If the primary cluster is a Rook Ceph cluster, the ceph-realm and ceph-zone-group resources on the pulling cluster will be identical to the ones on the primary cluster. 
+
+These resources just ensure the pulling cluster has the same ceph-realm and ceph-zonegroup resources as the primary cluster.
+
+If the primary Ceph cluster is not a Rook Ceph cluster, then the ceph-realm and ceph-zone-group resource on the pulling Ceph cluster will have the same name and config as the realm and zone group on the primary cluster.
+
+When these resources are created, neither the realm nor zone group is being recreated. 
+
+After these resources are automatically created the ceph-pull-realm resource is shut down.
+
+#### Steps
+1. At this point the Rook operator recognizes that a new ceph-pull-realm resource needs to be configured. The operator will start creating the resource to start the ceph-pull-realm.
+
+2. After these steps the admin should create:
+    1. A [ceph-zone](/design/ceph/ceph-zone.md) referring to the zone group the pullEndpoint is in.
+    2. A [object-store](/design/ceph/object-store.md) referring to the newly created ceph-zone.
+
+3. Once all of the resources steps #2 are started up, the operator will pull the realm on the rook Ceph cluster. If the realm pull succeeds, a ceph-realm and ceph-zone-group resource will be created.
```

Comment:
Why is it necessary to wait for the `ceph-zone` and `object-store` to be created to pull the realm?